.github/workflows/build-wsl.yaml

```diff
@@ -99,6 +99,19 @@ jobs:
         run: |
           mkdir -p ${{ env.workDir }}/wiki/
           git clone https://github.com/ubuntu/wsl.wiki ${{ env.workDir }}/wiki --depth 1
+      - name: Checkout ubuntu-archive-tools
+        shell: bash
+        run: |
+          mkdir -p ${{ env.workDir }}/uat
+          git clone https://git.launchpad.net/ubuntu-archive-tools ${{ env.workDir }}/uat --depth 1
+      - name: Prepare ISO Tracker credentials
+        working-directory: ${{ env.workDir }}
+        shell: bash
+        env:
+          ISOTRACKER_USERNAME: ${{ secrets.ISOTrackerUsername }}
+          ISOTRACKER_PASSWORD: ${{ secrets.ISOTrackerPassword }}
+        run: |
+          perl -pe 's/@([A-Z_]+)@/$ENV{$1}/eg' isotracker.conf.in >~/isotracker.conf
```

Comment:
Can’t we do that with sed? Because perl will make it harder to maintain :p
I would like the configuration to be store somewhere not in workdir if possible.

Reply:
I can look into doing it with sed ;p The perl version was straightforward as it could be written in one line, in case there's more credentials at some point. But that's rather unlikely!
As for the placement of the config file - I think this would require some changes to the isotracker module as by default it hard-codes this path for the config file. But I think this should be configurable, so I'll make the required modifications!